fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java

```diff
@@ -381,6 +387,20 @@ private static Map<String, NamedExpression> getColumnToOutput(
                 }
             }
         }
+        for (Column column : generatedColumns) {
```

Comment:
why cannot process in upper for loop?

Reply:
because the generated columns can use all ordinary columns, if processed in upper for loop, will lead to not found slot error

Reply:
It's the same reason for moving the processing of materialized columns down.